packages/rollup-plugin-html/src/minifyHtml.js

```diff
@@ -10,7 +10,7 @@ const defaultMinifyHTMLConfig = {
   useShortDoctype: true,
   minifyCSS: true,
   /** @param {string} code */
-  minifyJS: code => Terser.minify(code).code,
+  minifyJS: async code => (await Terser.minify(code)).code,
```

Comment:
The current test error reported by Circle CI is because the `minifyJS` option from the `html-minifier` dependency doesn't allow an async function. 😕 

Related: https://github.com/kangax/html-minifier/issues/900

Ping to @daKmoR @LarsDenBakker

Reply:
Also there is a new fork about `html-minifier` from the Terser team:

https://github.com/terser/html-minifier-terser